website/photos/services.py

```diff
@@ -173,11 +196,19 @@ def save_photo(photo_obj, file, filename):
     ):
         return False
 
+    rotation = photo_determine_rotation(image)
+    mirrored = photo_determine_mirrored(image)
```

Comment:
Unrelated to this specific piece of code but I think it might be possible to move all of this into Photo.save(). I'll probably look into that myself once I find some time